### DISCUSSION TITLE
Using yolov5 in project that uses 'opencv-contrib-python'

### DISCUSSION THREAD

**Discussion by nb-programmer** [fcakyon/yolov5-pip#176] [Q&A] [answered]
As the title says, my project has dependency for 'opencv-contrib-python' package as I need to use the tracker API (CSRT tracker), but yolov5 requires opencv-python, so installing it in my project installs both of them, which is bad and messes with the contrib package (It is recommended to not install both). So how should I handle this? How do I use both yolov5 and also have access to opencv-controb-python's API?

**Comment by fcakyon** [comment, marked as answer]
You can clone this repo, remove opencv in requirements file, then install from local 👍
